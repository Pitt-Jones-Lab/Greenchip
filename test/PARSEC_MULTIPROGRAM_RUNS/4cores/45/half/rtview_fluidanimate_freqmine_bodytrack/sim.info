{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/half/rtview_fluidanimate_freqmine_bodytrack',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_45_half_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/half/rtview_fluidanimate_freqmine_bodytrack',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmp73lSFB/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb9',
 'pin_version': 'unknown',
 'rusage': (22952.707657999999,
            819.95134800000005,
            205076,
            0,
            0,
            0,
            49372,
            1,
            0,
            61480,
            1344,
            0,
            0,
            0,
            64684446,
            6144672),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/half/rtview_fluidanimate_freqmine_bodytrack --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem45.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_45_half_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmp73lSFB/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/half/rtview_fluidanimate_freqmine_bodytrack/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 11325.562592029572,
 't_start': 1463065865.6388609,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1550999552L}
