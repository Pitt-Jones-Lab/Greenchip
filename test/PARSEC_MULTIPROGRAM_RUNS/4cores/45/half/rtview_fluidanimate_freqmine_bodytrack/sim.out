                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2936475065 |  2700273298 |  2675958155 |  1686196933
  Cycles                           |  2511960360 |  2511960360 |  2511960360 |  2511960360
  IPC                              |        1.17 |        1.07 |        1.07 |        0.67
  Time (ns)                        |   944346000 |   944346000 |   944346000 |   944346000
  Idle time (ns)                   |      110065 |   192334162 |      343432 |   311125871
  Idle time (%)                    |        0.0% |       20.4% |        0.0% |       32.9%
Branch predictor stats             |             |             |             |            
  num correct                      |   355305477 |   318244973 |   376786966 |   168660809
  num incorrect                    |    29675400 |    20509433 |    27021057 |    20548545
  misprediction rate               |       7.71% |       6.05% |       6.69% |      10.86%
  mpki                             |       10.11 |        7.60 |       10.10 |       12.19
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   407413313 |   350093957 |   342113253 |   192856121
    num misses                     |        4579 |         206 |         172 |         394
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |  1045851629 |   943956264 |  1128539994 |   512660757
    num misses                     |      503411 |     1181616 |      664020 |      507784
    miss rate                      |       0.05% |       0.13% |       0.06% |       0.10%
    mpki                           |        0.17 |        0.44 |        0.25 |        0.30
  L2 TLB                           |             |             |             |            
    num accesses                   |      507990 |     1181822 |      664192 |      508178
    num misses                     |      122708 |      330586 |      141054 |      119774
    miss rate                      |      24.16% |      27.97% |      21.24% |      23.57%
    mpki                           |        0.04 |        0.12 |        0.05 |        0.07
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   407413313 |   350093957 |   342113253 |   192856121
    num cache misses               |     5154915 |      993668 |      958052 |     1001640
    miss rate                      |       1.27% |       0.28% |       0.28% |       0.52%
    mpki                           |        1.76 |        0.37 |        0.36 |        0.59
  Cache L1-D                       |             |             |             |            
    num cache accesses             |  1045851629 |   943956264 |  1128539994 |   512660757
    num cache misses               |     5069032 |     6384015 |     6531173 |     5282621
    miss rate                      |       0.48% |       0.68% |       0.58% |       1.03%
    mpki                           |        1.73 |        2.36 |        2.44 |        3.13
  Cache L2                         |             |             |             |            
    num cache accesses             |    10586151 |     7839044 |     8213417 |     6627861
    num cache misses               |     2327830 |     2747137 |     2469468 |     1788140
    miss rate                      |      21.99% |      35.04% |      30.07% |      26.98%
    mpki                           |        0.79 |        1.02 |        0.92 |        1.06
DRAM summary                       |             |             |             |            
  num dram accesses                |    14198125 |           0 |           0 |           0
  average dram access latency (ns) |      115.13 |         inf |         inf |         inf
  average dram queueing delay      |      115.13 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     1531346 |     1488688 |     1837292 |     1248216
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
